interfaces/builtin/hardware_observe.go

```diff
@@ -0,0 +1,66 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+// http://bazaar.launchpad.net/~ubuntu-security/ubuntu-core-security/trunk/view/head:/data/apparmor/policygroups/ubuntu-core/16.04/log-observe
+const hardwareObserveConnectedPlugAppArmor = `
+# Description: This interface allows for getting hardware information
+# from the system, as is needed by checkbox on snappy.  This is reserved for OS snap.
+# Usage: reserved
+
+#include <abstractions/base>
+
+/bin/udevadm ixr,
+/bin/lsblk ixr,
+/usr/sbin/dmidecode ixr,
+/usr/bin/lsusb ixr,
```

Comment:
Is `lsusb` actually in the image? I would say that after having the permissions granted by this interface it would be better to ship the software that has no daemon component (not udev) as a snap itself.